test/metabase/api/collection_test.clj

```diff
@@ -1807,3 +1808,19 @@
                                        (assoc (graph/graph)
                                               :groups {group-id {default-a :write, currency-a :write}}
                                               :namespace :currency)))))))))
+
+(deftest list-collections-instance-analytics-test
+  (mt/test-drivers #{:postgres :h2 :mysql}
+    (audit-db-test/with-audit-db-restoration
+      (audit-db/ensure-audit-db-installed!)
+      (testing "Intsance Analytics Collection should be the last collection."
```

Comment:
Maybe create at least one temp collection to make sure that it's not last just because it's the only one? I guess there will implicitly always be the root collection but it might be nice to have a sanity check.